### PR TITLE
TUT-101: Added new empty project

### PR DESCRIPTION
**Pull request by theyogeshrathod**
Added all the auto-generated files as well.

### CODE REVIEW

**Review by yogeshrr0407 — COMMENTED** [review]
Remove all .gradle files

**Review by yogeshrr0407 — CHANGES_REQUESTED** [review]
Remove all .idea/ files

File: app/app.iml [hunk 1]
```diff
@@ -0,0 +1,163 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (yogeshrr0407) [inline app/app.iml:1]: Remove this file

File: app/proguard-rules.pro [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# Add project specific ProGuard rules here.
```
Comment (yogeshrr0407) [inline app/proguard-rules.pro:1]: Remove this file

**Comment by theyogeshrathod** [comment]
Closing pull-request as a lot of unnecessary files are pushed in.

File: app/app.iml (hunk 1 above)
Comment (theyogeshrathod) [reply app/app.iml]: img_1552

**Pull request closed without merging**